src/shared/components/CallToActionButton/CallToActionButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react'
+
+import { SvgGlyphChevronRight } from '@/shared/icons'
+import { getLinkPropsFromTo } from '@/utils/button'
+
+import { BodyWrapper, ContentWrapper, IconWrapper, StyledContainer } from './CallToActionButton.style'
+import { CallToActionButtonProps } from './CallToActionButton.types'
+
+export const CallToActionButton: FC<CallToActionButtonProps> = ({
+  to,
+  icon,
+  onClick,
+  colorVariant = 'blue',
+  label,
+}) => {
+  const linkProps = getLinkPropsFromTo(to)
+
+  return (
+    <StyledContainer {...linkProps} onClick={onClick} colorVariant={colorVariant}>
```

Comment:
![image](https://user-images.githubusercontent.com/51168865/127136420-552e6ad8-fdef-4c6f-a1f3-ad612c2c4be0.png)
To fix this you could use `shouldForwardProp` like I did https://github.com/Joystream/atlas/pull/1016/files